### ISSUE TITLE
"Can't read from hidraw, is the program running as root?"

### ISSUE THREAD

**Issue by GCAT01** [linux-surface/iptsd#181]
Just installed linux-surface (you guys rock!) and I have been having trouble with the touchscreen/gestures. Through my various searches for troubleshooting, I came across the command `iptsd-find-hidraw` and I received the following:
```
[warning] Can't read from /dev/hidraw#, is the program running as root?
[info] No devices found
```
where "#" is a digit from 0-4.

I have yet to see this message listed anywhere as an output of any iptsd command so I am asking for some guidance.

**Comment by GCAT01** [comment]
Apparently
```
genericuser3@BeyondBright:~$ ps -elf|grep hidraw
4 S root         709       1  0  80   0 -  1777 -      13:15 ?        00:00:00 /usr/bin/iptsd /dev/hidraw4
0 S generic+    2464    2396  0  80   0 -  2286 pipe_r 13:24 pts/0    00:00:00 grep --color=auto hidraw
```

**Comment by GCAT01** [comment]
In my attempts to fix the problem, I decided to try a firmware update as I was betting the previous owner had never done one. I tried to use the script provided by linux-surface but ran into UEFI secure boot issues. Turns out the MOK key never got registered despite going through the process after linux-surface installation. Installed the key via command-line and it is now registered.

Was still running into a problem trying to install the firmware, however, and after banging my head against a wall for a while, I decided what little touch control I had was going to have to do. On a hunch, I switched back from the Wiacom environment to default Mint Cinnamon and... everything seems to work. The cursor follows my finger, clicks on touch, I can pinch zoom. Apparently it just needed several restarts to get everything in order.
